rundeckapp/src/main/groovy/org/rundeck/app/data/providers/GormProjectDataProvider.groovy

```diff
@@ -80,13 +81,14 @@ class GormProjectDataProvider implements RundeckProjectDataProvider {
     }
 
     @Override
-    @CompileStatic(TypeCheckingMode.SKIP)
-    Collection<String> getFrameworkProjectNames() {
-        def c = Project.createCriteria().list {
-            projections {
-                property "name"
-            }
-        }
+    RdProject findByNameAndState(String name, RdProject.State state) {
+        RdProject project = projectDataService.getByName(name)
```

Comment:
Using data services instead of criteria queries causes weird data problems. There seems to be a bug in the Gorm internals that isn't fixed, so we should stay away from data services until it's resolved.